.github/workflows/sycl-post-commit.yml

```diff
@@ -103,7 +103,11 @@ jobs:
       && success()
       && github.repository == 'intel/llvm'
     uses: ./.github/workflows/sycl-windows-build.yml
-
+    with:
+      compiler: icx
+      build_configure_extra_args: --cmake-opt=-DCMAKE_C_FLAGS="/clang:-Wno-nonportable-include-path /clang:-Wno-cast-function-type-mismatch" --cmake-opt=-DCMAKE_CXX_FLAGS="/clang:-Wno-nonportable-include-path /clang:-Wno-cast-function-type-mismatch" --cmake-opt="-DCMAKE_EXE_LINKER_FLAGS=/manifest:no" --cmake-opt="-DCMAKE_MODULE_LINKER_FLAGS=/manifest:no" --cmake-opt="-DCMAKE_SHARED_LINKER_FLAGS=/manifest:no" 
```

Comment:
I think this should go to CMake instead, but it's been a tremendous amount of work already, so I won't insist :)

Reply:
agree but yeah please allow me some time regain my sanity, will do in follow up

Reply:
made https://github.com/intel/llvm/issues/16363 for follow up